src/sections/blog-post.liquid

```diff
@@ -0,0 +1,155 @@
+{%- comment -%}
+  This is a required section for the Shopify Theme Store.
+  It is loaded when you add "Blog Posts" section on the Theme Editor.
+
+  We are showing a preview of what the blog post looks like in the preview.
```

Comment:
Not sure what you mean by this.

Reply:
I removed it. After reading it, it doesn't make sense 😢